creator-node/src/routes/files.js

```diff
@@ -303,6 +303,20 @@ const getCID = async (req, res) => {
             `${logPrefix} No valid file found for provided CID`
           )
         )
+      } else if (queryResults.type === 'dir') {
```

Comment:
direct copy of old version of this code from https://github.com/AudiusProject/audius-protocol/blob/76f7cf714d/creator-node/src/routes/files.js#L208